src/content/en/tools/workbox/index.md

```diff
@@ -0,0 +1,177 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Service Worker Libraries.
+
+{# wf_published_on: 2015-01-01 #}
+{# wf_updated_on: 2017-05-16 #}
+
+# Workbox {: .page-title }
+
+Use [Workbox](https://workboxjs.org/) to simplify your development by making it
+easy to take advantage of powerful service worker features, eliminate boilerplate code, and automate service worker generation.
+
+<figure class="attempt-right">
+  <img src="/web/tools/workbox/thumb.png" alt="Workbo logo">
+</figure>
+
+Workbox is a collection of loosely-coupled libraries and tools, that focus on
+different service worker features and use-cases. The two core tools that Workbox provides are:
+
+**workbox-sw&mdash;**A service worker library to make managing fetch requests
+and caching as easy as possible.
+
+**workbox-cli&mdash;**A command-line tool to generate a service worker and a
+file manifest, that makes use of the workbox-sw module to become a fully-functioning service worker.
+
+These two high-level tools are built up from a number of lower-level modules
+that can be used independently or mixed-and-matched:
+
+**workbox-build&mdash;**A module that gan be used to generate a file manifest or
+service worker, that can then be used with `workbox-sw`. The `workbox-build`
+module makes it easy to generate a service working using Gulp, Webpack, or any
+other build tool you might use. This is the core of the Workbox CLI.
+
+**workbox-runtime-caching&mdash;**A service worker helper library that
+implements various runtime caching strategies.
+
+**workbox-cache-expiration&mdash;**A service worker helper library that expires
+cached responses based on age or a maximum number of entries.
+
+Workbox also includes modules for other common service worker use-cases:
+
+**workbox-google-analytics&mdash;**A service worker helper library to retry
+offline Google Analytics requests when a connection is available.
+
+**workbox-background-sync&mdash;**A generic helper library that queues failed
+requests and uses the Background Sync API to replay those requests when the user
+comes back online.
+
+Workbox is the next generation of our service worker libraries and tools,
+previously called `sw-toolbox` and `sw-precache`. If you were familiar with
+those tools, Workbox will be very familiar as well. Workbox is the result of
+redesigning these tools to be more modular, flexible, and extensible.
```

Comment:
I'm worried that this paragraph makes Wrokbox sound like Service Worker Libraries 2.0. It isn't literally that. @addyosmani